src/main/java/pl/antonina/tasks/allegro/accessToken/AccessTokenConfig.java

```diff
@@ -0,0 +1,24 @@
+package pl.antonina.tasks.allegro.accessToken;
+
+import okhttp3.Credentials;
+import okhttp3.FormBody;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+class AccessTokenConfig {
+
+    private final String clientId = "2d28293921ae472c9d82d99b2d9268eb";
+    private final String clientSecret = "k0GW7MrzsCaIvN3o8NkPQx1GdsjBhEpFZG1dzYaj0NdFP8B27vHTAA1K5SRIXTys";
```

Comment:
We ca go even further and move there to properties file! :) https://www.baeldung.com/properties-with-spring
It would allow you not to change the code (and deploy the app) every time these values change (and they are not under your control, they can change without your knowledge).